source/__tests__/integration/import-profile.test.js

```diff
@@ -2,8 +2,7 @@
 const path = require('path')
 const pupExpect = require('./jestPuppeteerHelper')
 
-describe('imports and edits a v0.0.2 profile from a json file', () => {
-
+describe('imports valid profile without included schema url from json file', () => {
```

Comment:
from the perspective of the profile editor, v0.0.2 and v0.0.3 are the same -- they are both for profiles / resource templates that have no explicit schema indicated.  The difference is that v0.0.3 doesn't allow whitespace in ids, which is desirable for trellis.  That check shows up in the sinopia_editor, not here.